templates/zerver/for/communities.md

```diff
@@ -0,0 +1,137 @@
+>  “Choosing Zulip over Slack as our group chat is one of the best
+>  decisions we’ve ever made. Zulip makes it easy for our community of
+>  1000 Recursers around the world to stay involved, even years after
+>  their batches finish. No other tool has a user experience that
+>  [scales to a community of our
+>  size](https://www.recurse.com/blog/112-how-rc-uses-zulip).”
+>
+> — Nick Bergson-Shilcock, founder and CEO, Recurse Center
+
+Zulip is designed to help thoughtful people work on difficult problems
+together, whether they work from a shared office or from all over the
+world. Zulip offers an ideal platform for communities of all types,
+including open-source projects, research collaborations, volunteer
+organizations, and other groups of people who share a common pursuit.
+
+The Zulip core developers have decades of combined experience leading
+and growing open source communities, and we use Zulip to fashion the
+day-to-day experience of being a part of our project. No other chat
+product comes close to Zulip in facilitating contributor engagement,
+facilitating inclusion, and making efficient use of everyone’s time.
+
+&nbsp;
+
+<blockquote class="twitter-tweet"><p lang="en" dir="ltr">When we made the switch to <a href="https://twitter.com/zulip?ref_src=twsrc%5Etfw">@zulip</a> a few months ago for chat, never in my wildest dreams did I imagine it was going to become the beating heart of the community, and so quickly. It&#39;s a game changer. 🧑‍💻🗨️👩‍💻</p>&mdash; Dan Allen (@mojavelinux) <a href="https://twitter.com/mojavelinux/status/1409702273400201217?ref_src=twsrc%5Etfw">June 29, 2021</a></blockquote>
+&nbsp;
+
+If you haven’t read [Why Zulip](https://zulip.com/why-zulip), read
+that first. The challenges with the Slack/Discord/IRC model discussed
+there are even more important for open communities:
```

Comment:
This isn't new in this commit, but I'll take the opportunity to voice my opinion 🙂

What was the thought process behind this line? To me personally, something like "*read that first*" seems kinda rude (especially since this page is pretty understandable even without reading the "Why Zulip" page). I think this should be replaced with something like:
```quote
The challenges with the Slack/Discord/IRC model, which we talk about in our [Why Zulip](link) page,
are even more important for open communities:
```

Reply:
I agree we should change that text; it doesn't read right to me either.  The goal of this PR is to remove the semi-duplicate copies we have of that content, so I don't think that blocks this work.

@alya FYI to put this in your TODO list.